### ISSUE TITLE
Compare SPSS vs R outputs 

### ISSUE THREAD

**Issue by Jennit07** [Public-Health-Scotland/source-linkage-files#475]
Still to do and update the spreadsheet on teams: 

- [x] Care Homes (year specific)
- [ ] ALL Care Homes (on hold, still WIP)
- [ ] GP OOH (on hold - waiting on approved PR)
- [ ] Delayed Discharges 
- [x] DN
- [x] Home Care 
- [x] A & T 
- [x] SDS
